src/libraries/System.Net.Security/src/System/Security/Authentication/ExtendedProtection/ExtendedProtectionPolicy.cs

```diff
@@ -149,7 +149,7 @@ public static bool OSSupportsExtendedProtection
             get
             {
                 // .NET Core is supported only on Win7+ where ExtendedProtection is supported.
-                return true;
+                return OperatingSystem.IsWindows();
```

Comment:
This is controversial. I can change it to `!OperatingSystem.IsBrowser()` to be on the safe side and behave like older .NET versions did. However, the reality is that it's not really implemented on Linux/macOS either. Some bits and pieces may have effect (eg. SPN validation for NTLM Negotiation) but others are blatantly ignored (https://github.com/dotnet/runtime/pull/87930/files#r1263108882).

Reply:
It's better to say it's not supported when there's known gaps than to say it's supported and have the consumer believe ExtendedProtection is being used when it's actually not. It's off by default, so if you explicitly turn it on, it should actually do the expected thing.

Reply:
I think this is OK. I feel it is better to claim `false` if the functionality is not clear or complete.